### PR TITLE
Added FLCO-FLYCOLORF7_AIO targets

### PR DESCRIPTION
**Pull request by tobeornottobe81**
Added FLCO_FLYCOLORF7_AIO targets

### CODE REVIEW

File: configs/default/FLCO-FLYCOLORF7_AIO.config [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+set baro_i2c_device = 1
+set adc_device = 3
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 170
+set beeper_inversion = ON
+set beeper_od = OFF
+set pid_process_denom = 4
```
Comment (etracer65) [inline configs/default/FLCO-FLYCOLORF7_AIO.config:130]: This is certainly wrong. There would be no reason to run the PID loop at 2khz on a F7. Remove this line.

File: configs/default/FLCO-FLYCOLORF7_AIO.config [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+set dashboard_i2c_bus = 1
+set pinio_box = 40,41,255,255
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180FLIP
+set gyro_1_align_pitch = 1800
+set gyro_1_align_yaw = 1800
+set align_board_yaw = 180
+set align_board_pitch = 180
```
Comment (etracer65) [inline configs/default/FLCO-FLYCOLORF7_AIO.config:141]: Setting board alignments in the target config doesn't make any sense. It's as if you're telling the user to install the board incorrectly and not match the indicators on the board or documentation. The board alignments should not be set and instead the gyro sensor alignments should be corrected based on the expected board installation orientation.

File: configs/default/FLCO-FLYCOLORF7_AIO.config (hunk 1 above)
Comment (tobeornottobe81) [reply configs/default/FLCO-FLYCOLORF7_AIO.config:130]: thank you! i will correct it

File: configs/default/FLCO-FLYCOLORF7_AIO.config (hunk 2 above)
Comment (tobeornottobe81) [reply configs/default/FLCO-FLYCOLORF7_AIO.config:141]: OK, thank you for your guidance

**Comment by etracer65** [comment]
So should we close this pull request? You have removed all the content.

**Pull request closed without merging**